esignet-injiweb.properties

```diff
@@ -447,9 +474,16 @@ mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.registry-search-url=http:
 mosip.esignet.authenticator.sunbird-rc.kba.entity-id-field=osid
 
 mosip.esignet.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
-mosip.esignet.vciplugin.sunbird-rc.supported-credential-types=InsuranceCredential
+mosip.esignet.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential
+mosip.esignet.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+#mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=https://raw.githubusercontent.com/mosip/mosip-config/develop/insurance-credential.json
+mosip.esignet.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.template-url=https://raw.githubusercontent.com/challabeehyv/mimoto-config/main/InsuranceConfig.json
+mosip.esignet.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
+mosip.esignet.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-id=did:schema:0d10a2cf-94de-4ffc-b32c-4f1a61ee05ba
+mosip.esignet.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-version=1.0.0
+
 mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
-mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/insurance-credential.json
+mosip.esignet.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=https://raw.githubusercontent.com/challabeehyv/mimoto-config/main/InsuranceConfig.json
```

Comment:
Do we need this change ? Can we rather update the existing json ?
https://github.com/mosip/mosip-config/blob/develop/insurance-credential.json

Reply:
Above comment applies for the LifeInsuranceCredential also

Reply:
updated the insurance-credential.json with new context path to support both credential types

Reply:
Because of the type we actually need 2 different json files, instead of pointing to the outside github, can we include the life insurance as a separate file in this PR and point to that ? We can call it life-insurance-credential.json

Reply:
so there will be two json files one for template and other one for context, and this PR should have both files
is my understanding correct?

Reply:
added the new file  life-insurance-credential.json